src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,38 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+
+        ManufacturerService manufacturerService =
+                 (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        Manufacturer czechManufacturer =
+                new Manufacturer("Skoda", "Czech");
+        manufacturerService.create(czechManufacturer);
+        System.out.println(manufacturerService.get(czechManufacturer.getId()));
+        czechManufacturer.setName("Czech republic");
+        System.out.println(manufacturerService.update(czechManufacturer));
+        manufacturerService.delete(czechManufacturer.getId());
+        List<Manufacturer> manufacturers = manufacturerService.getAll();
+        manufacturers.forEach(System.out::println);
```

Comment:
```suggestion
         manufacturerService.getAll().forEach(System.out::println);
```

Reply:
I know that, i just created list to cover method getAll for list

Reply:
okay